frontend/public/bin.html

```diff
@@ -43,15 +43,14 @@ <h2>Requests for <span id="bin-header">[bin]</span></h2>
             
         </span>
       </main>
-      <!-- the fact that each span has a different class might not be necessary, they could potentially all share a single class such as "catcher-data" -->
```

Comment:
it's not necessary - it should be multiple classes on the sections that have different text attributes.